Resources/projects/lean-engine-branches/change-branches.php

```diff
@@ -1,28 +1,11 @@
-<?php
-$getChangeBranchesText = function($isDesktopDocs, $openProjectLink, $createProjectLink)
-{        
-    echo "
 <p>Follow these steps to change the LEAN engine branch that runs your backtests and live trading algorithms:</p>
-
 <ol>
-    <li><a href='{$openProjectLink}'>Open a project</a>.</li>
-    ";
-  
-    if ($isDesktopDocs)
-    {
-        echo "
-<li>In the left navigation menu, click the <img class='inline-icon' src='https://cdn.quantconnect.com/i/tu/vscode-qc-icon.jpg'> <span class='icon-name'>QuantConnect</span> icon.</li>
-        "; 
-    }
-  
-    echo "
+    <li><a href='<?=$openProjectLink?>'>Open a project</a>.</li>
+    <?php if (!$cloudPlatform) { ?><li>In the left navigation menu, click the <img class='inline-icon' src='https://cdn.quantconnect.com/i/tu/vscode-qc-icon.jpg'> <span class='icon-name'>QuantConnect</span> icon.</li><?php } ?>
```

Comment:
Please invert the statement and use inline statements to reduce nesting and avoid its applying to research/engine urls

`<?=$localPlatform ? "<li>In the left navigation menu, click the <img class='inline-icon' src='https://cdn.quantconnect.com/i/tu/vscode-qc-icon.jpg'> <span class='icon-name'>QuantConnect</span> icon.</li>" : "" ?>`